### PR TITLE
Constant time Solution of the Missing range Program

### PR DESCRIPTION
**Pull request by shoaibrayeen**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [X] **If creating a new file :**
  - [X] added links to it in the README files ?
  - [X] included tests with it ?
  - [X] added description (overview of algorithm, time and space compleixty, and possible edge case) in docstrings ?

- [X] **if done some changes :**
  - [X] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [X] **other**
 - Added description about general View

### CODE REVIEW

File: algorithms/arrays/missing_ranges.py [hunk 1]
```diff
@@ -20,3 +21,43 @@ def missing_ranges(arr, lo, hi):
         res.append((start, hi))     # append remainder to list
 
     return res
+'''
+
+# May be this would be good approach
+
+def missingRanges(arr , low , high):
```
Comment (keon) [inline algorithms/arrays/missing_ranges.py]: please use snake case
